packages/client/src/Authentication.ts

```diff
@@ -40,7 +39,7 @@ export const createAuthentication = (config: Pick<StrictStreamrClientConfig, 'au
         return createPrivateKeyAuthentication(normalizedPrivateKey)
     } else if ((config.auth as ProviderAuthConfig)?.ethereum !== undefined) {
         const ethereum = (config.auth as ProviderAuthConfig)?.ethereum
-        const provider = new Web3Provider(ethereum)
+        const provider = new BrowserProvider(ethereum as any)
```

Comment:
Would it make sense to use `Eip1193Provider` as the type of `ProviderAuthConfig#ethereum`?